xbmc/network/websocket/WebSocketManager.cpp

```diff
@@ -49,7 +49,7 @@ CWebSocket* CWebSocketManager::Handle(const char* data, unsigned int length, std
   {
     CLog::Log(LOGINFO, "WebSocket: missing Sec-WebSocket-Version");
     CHttpResponse httpResponse(HTTP::Get, HTTP::BadRequest, HTTP::Version1_1);
-    response = response = httpResponse.Create();;
+    response = httpResponse.Create();;
```

Comment:
@domielias there 2 `;;` at the end of this line. can you please remove one of them and then
`git commit -a --amend --no-edit` and a force-push?
noticed it in the diff here https://jenkins.kodi.tv/job/BuildMulti-PR/18460/artifact/PR18725.diff

Reply:
@howie-f Sure, i made it. Check it now.

Reply:
:+1: